mapper/tedge_mapper/src/component.rs

```diff
@@ -0,0 +1,7 @@
+use async_trait::async_trait;
+use tedge_config::TEdgeConfig;
+
+#[async_trait]
+pub trait TEdgeComponent {
```

Comment:
Having the trait to automatically implement Debug would make it easier for the future debug.

```suggestion
pub trait TEdgeComponent: std::fmt::Debug {
```

Reply:
I'll do that.